_posts/2016-03-14-Containers-all-the-way-through.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+name: containers-all-the-way-through
+title: Containers all the way through...
+date: 2016-03-14 00:00:00 +00:00
+author: Mani Sarkar
+image:
+    src: /assets/img/custom/blog/2016-03-14-Containers-all-the-way-through/cover-image.png
+    attribution:
+        text: World Inside Drops by Joni Niemelä
+        href: https://www.flickr.com/photos/nitrok/4643690435/in/photolist-85m8oc-dJdHdG-rCxKY5-cSZWGm-aoSk6R-cTJyMG-AGapwu-8V9PDD-8RZxMr-9ZSVdH-8BGYS9-iENx1g-nNVtCZ-ezCSP4-4awLWU-bGvJHx-7izcMQ-5Us4oC-5fS7ye-np9Kag-nxQW1H-97rFRm-7CDVSn-amdWzY-6yydhM-7sCqL6-9hCdZ1-8o271S-pnwdfw-dQ7qGx-9RtUs9-5iXWSw-7fnmbc-7BMPf9-rAWL1e-a9DtWV-dUWBgE-prbj6-ioWvpv-6w6P2v-aDL49E-9DZy8z-kEKm9A-5m6nao-91AkmW-91nNKW-7rLbuB-aYn55V-py32i8-bWcJBy
+canonical:
+    name: my personal blog
+    href: https://neomatrix369.wordpress.com/2016/03/13/containers-all-the-way-through/
+tags:
+- bare metal system
+- cgroups
+- container
+- container system
+- executable
+- hypervisor 
+- kernel
+- namespaces 
+- operating system
+- process 
+- shared operating system
+- shared resources
+- virtual hardware
+- virtual machine 
+- virtual system
+- vm
```

Comment:
Not sure you need that many tags. Tags are here to group articles together, and Codurance has very few virtualization-related articles, so your article would be alone in most tags. The goal is rather than if people click on those tags, they find similar article.

Also, this amount of tags leaves a lot of white space on the blog front page.

![image](https://cloud.githubusercontent.com/assets/1684420/13760799/e62354b2-ea2c-11e5-8bc7-ab19ae0e4a02.png)